lighthouse-core/report/templates/report-template.html

```diff
@@ -177,3 +176,13 @@ <h1 class="menu__header-title">Lighthouse</h1>
     </footer>
   </section>
 </div>
+
+{{#if scripts }}
+<script>
+  window.lhresults = {{{lhresults}}};
```

Comment:
I also prefer putting the lhresults in a non-script element. How about `<template>`?  
TBH when i was first trying out the lhresults, it created some html elements from various descriptions. I think those things are addressed now, but i feel safer not throwing fairly arbitrary text into a script tag.